support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.js

```diff
@@ -110,6 +112,10 @@ const applyBillingAddressRules = (fields: FormFields, addressType: AddressType):
       fields.country === 'CA' ? `Please select a ${addressType} province/territory.` : `Please select a ${addressType} state.`,
     ),
   },
+  {
+    rule: checkpostCodeLength(fields.postCode),
+    error: formError('postCode', `Postcode should not be longer than 20 characters.`),
```

Comment:
looks like on the other address validation errors we also include `${addressType}` in the message, so the summary points out which it is, so best to add that.

Reply:
Done now.